guides/source/active_record_querying.md

```diff
@@ -714,40 +714,40 @@ To retrieve records from the database in a specific order, you can use the [`ord
 For example, if you're getting a set of records and want to order them in ascending order by the `created_at` field in your table:
 
 ```ruby
-Customer.order(:created_at)
+Book.order(:created_at)
 # OR
-Customer.order("created_at")
+Book.order("created_at")
 ```
 
 You could specify `ASC` or `DESC` as well:
 
 ```ruby
-Customer.order(created_at: :desc)
+Book.order(created_at: :desc)
 # OR
-Customer.order(created_at: :asc)
+Book.order(created_at: :asc)
 # OR
-Customer.order("created_at DESC")
+Book.order("created_at DESC")
 # OR
-Customer.order("created_at ASC")
+Book.order("created_at ASC")
 ```
 
 Or ordering by multiple fields:
 
 ```ruby
-Customer.order(orders_count: :asc, created_at: :desc)
+Book.order(title: :asc, created_at: :desc)
 # OR
-Customer.order(:orders_count, created_at: :desc)
+Book.order(:title, created_at: :desc)
 # OR
-Customer.order("orders_count ASC, created_at DESC")
+Book.order("title ASC, created_at DESC")
 # OR
-Customer.order("orders_count ASC", "created_at DESC")
+Book.order("title ASC", "created_at DESC")
 ```
 
 If you want to call `order` multiple times, subsequent orders will be appended to the first:
 
 ```irb
-irb> Customer.order("orders_count ASC").order("created_at DESC")
-SELECT * FROM customers ORDER BY orders_count ASC, created_at DESC
+irb> Book.order("title ASC").order("created_at DESC")
+SELECT * FROM authors ORDER BY title ASC, created_at DESC
```

Comment:
```suggestion
SELECT * FROM books ORDER BY title ASC, created_at DESC
```

Reply:
Thanks @kamipo. I've commited the suggestion.